ui/oopsyraidsy/oopsy_summary_list.ts

```diff
@@ -82,7 +82,7 @@ export class OopsySummaryTable implements MistakeObserver {
 
   OnMistakeObj(m: OopsyMistake): void {
     const longName = m.name ?? m.blame;
-    if (!longName)
+    if (longName === undefined || longName.length === 0)
```

Comment:
Same comment as in the other file, but I think it'd make more sense to just have it be `???` here rather than skipping?